dev-python/axolotl-curve25519/axolotl-curve25519-0.4.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_P="python-${P}"
```

Comment:
This looks backwards. Define `MY_P` as `MY_PN-PV`.

Reply:
Ok, I changed it, to make it better, in both eBuilds.